ts/features/fastLogin/store/selectors/index.ts

```diff
@@ -1,16 +1,28 @@
 import { createSelector } from "reselect";
 import { uniqWith, isEqual } from "lodash";
 import { backendStatusSelector } from "../../../../store/reducers/backendStatus";
-import { fastLoginEnabled } from "../../../../config";
+import { fastLoginBypassOptInt, fastLoginEnabled } from "../../../../config";
 import { GlobalState } from "../../../../store/reducers/types";
 import { isPropertyWithMinAppVersionEnabled } from "./../../../../store/reducers/backendStatus";
 
+const fastLoginOptInSelector = (state: GlobalState) =>
+  state.features.loginFeatures.fastLogin.optIn;
+
+export const isFastLoginOptinEnabledSelector = createSelector(
+  fastLoginOptInSelector,
+  optIn => {
+    if (fastLoginBypassOptInt) {
+      return true;
+    }
+    return optIn.fastLoginEnabled;
+  }
+);
 /**
  * return the remote config about FastLogin enabled/disabled
  * based on a minumum version of the app.
  * if there is no data, false is the default value -> (FastLogin disabled)
  */
-export const isFastLoginEnabledSelector = createSelector(
+export const isFastLoginFFEnabled = createSelector(
```

Comment:
I would leave the old FF and ad another FF. This is the scenario:
- The old one allow us to enable/disable locally the LV as a whole feature (this allow us to deploy beta builds).
- The new one, maybe along with a sibling remote one, allow us to enable/disable only the opt-in sub feature.

What do you think?